docs/DafnyRef/Attributes.md

```diff
@@ -218,7 +217,26 @@ function g(y:int, b:bool) : bool
 }
 ```
 
-### 11.2.3. `{:axiom}` {#sec-axiom}
+### 11.2.3. `{:autoRevealDependencies k}` {#sec-autorevealdependencies}
+When setting `--default-function-opacity` to `autoRevealDependencies`, the `{:autoRevealDependencies k}` attribute can be set on methods and functions to make sure that only function dependencies of depth `k` in the call-graph or less are revealed automatically. As special cases, one can also use `{:autoRevealDependencies false}` (or `{:autoRevealDependencies 0}`) to make sure that no dependencies are revealed, and `{:autoRevealDependencies true}` to make sure that all dependencies are revealed automatically.
```

Comment:
Throughout the document, is the argument to `--default-function-opacity` written `AutoRevealDependencies` or `autoRevealDependencies`?

Reply:
That is actually an important point. Technically both work, but I am not sure which the canonical choice should be. The command-line documentation which picks the argument names (I think) from the underlying enum datatype, has everything capitalized (because in the codebase that is the norm), whereas in the command-line I think the norm is to keep the first letter lowercase in camelCase. What do you think?

Reply:
For me, on the CLI, it should start with a lowercase if possible

Reply:
Done. Uniformly changed it to "autoRevealDependencies".